### PR TITLE
[SDK-493] Payment Pay anyone

### PR DESCRIPTION
**Pull request by dipeshfrollo**
[SDK-493] 

[SDK-493]: https://frollo.atlassian.net/browse/SDK-493

### CODE REVIEW

**Review by MaherKSantina — APPROVED** [review]

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/Model/API/Payments/APIPayAnyoneRequest.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    
+    let accountHolder: String
+    let accountNumber: String
+    let amount: String
+    let bsb: String
+    let description: String?
+    let paymentDate: String?
+    let reference: String?
+    let sourceAccountId: Int64
```
Comment (nickfrollo) [inline Sources/Model/API/Payments/APIPayAnyoneRequest.swift:39]: Can we capitalise ID for consistency with the rest of the project?

**Review by nickfrollo — CHANGES_REQUESTED** [review]
Great job!

We may wish to cache Payment response as in the future there will be a way to fetch scheduled payments. I wouldn't worry about this for now though.

File: Sources/Model/API/Payments/APIPayAnyoneRequest.swift (hunk 1 above)
Comment (dipeshfrollo) [reply Sources/Model/API/Payments/APIPayAnyoneRequest.swift:39]: Will update. Thanks

**Comment by dipeshfrollo** [comment]
@nickfrollo yes we will do the caching when we implement GET /payments. BTW responses of /payments/bpay, /payments/payanyone and /payments/transfer are different so each will have different models (uncached)

**Comment by MaherKSantina** [comment]
So I guess we have 2 choices to cache the data:
1. Do it similar to messages where there is a table for each type and a main one
2. Cache everything in one table and have a different class for each type, and when we fetch the payments from the table we map the fields to the corresponding type

Personally I prefer option 2 because it doesn't require multiple tables in the database and we can get the data from one place

**Comment by dipeshfrollo** [comment]
3. Only cache for GET /payments 😄  . Why should we cache these responses anyway?

**Comment by MaherKSantina** [comment]
haha right! I'm happy with that

**Review by nickfrollo — APPROVED** [review]

**Pull request merged**